tenders/common/mft.c

```diff
@@ -142,6 +142,9 @@ const char *mft_type_to_string(mft_type_t type)
         case MFT_RESERVED_FIRST:
             return "RESERVED_FIRST";
         default:
-            assert(false);
+            /* This is used only to pretty print the content of the manifest so
+               a generic error is probably good enough, if only partially
+               informative */
+            return "<undefined type>";
```

Comment:
Would it make sense to "print" the integer value of `type` in the string? So the developer of a new device / module has a better chance to figure out what went wrong. Then again if someone is writing a module they have a better chance of knowing to touch this part of the internals...